libmscore/marker.cpp

```diff
@@ -29,15 +29,15 @@ static const ElementStyle markerStyle {
 
 //must be in sync with Marker::Type enum
 const MarkerTypeItem markerTypeTable[] = {
-      { Marker::Type::SEGNO    , QT_TRANSLATE_NOOP("markerType", "Segno")          },
-      { Marker::Type::VARSEGNO , QT_TRANSLATE_NOOP("markerType", "Segno variation")},
-      { Marker::Type::CODA     , QT_TRANSLATE_NOOP("markerType", "Coda")           },
-      { Marker::Type::VARCODA  , QT_TRANSLATE_NOOP("markerType", "Varied coda")    },
-      { Marker::Type::CODETTA  , QT_TRANSLATE_NOOP("markerType", "Codetta")        },
-      { Marker::Type::FINE     , QT_TRANSLATE_NOOP("markerType", "Fine")           },
-      { Marker::Type::TOCODA   , QT_TRANSLATE_NOOP("markerType", "To Coda")        },
-      { Marker::Type::TOCODASYM, QT_TRANSLATE_NOOP("markerType", "To Coda")        },
-      { Marker::Type::USER     , QT_TRANSLATE_NOOP("markerType", "Custom")         }
+      { Marker::Type::SEGNO    , QT_TRANSLATE_NOOP("markerType", "Segno")           },
+      { Marker::Type::VARSEGNO , QT_TRANSLATE_NOOP("markerType", "Segno variation") },
+      { Marker::Type::CODA     , QT_TRANSLATE_NOOP("markerType", "Coda")            },
+      { Marker::Type::VARCODA  , QT_TRANSLATE_NOOP("markerType", "Varied coda")     },
+      { Marker::Type::CODETTA  , QT_TRANSLATE_NOOP("markerType", "Codetta")         },
+      { Marker::Type::FINE     , QT_TRANSLATE_NOOP("markerType", "Fine")            },
+      { Marker::Type::TOCODA   , QT_TRANSLATE_NOOP("markerType", "To Coda")         },
+      { Marker::Type::TOCODASYM, QT_TRANSLATE_NOOP("markerType", "To Coda (Symbol)")},
```

Comment:
To be able to distinguish the 2 Codas, esp. for visually impaired users (and to appease the tst_palette mtest)